data/changelog/ocaml/2024-05-13-ocaml-5.2.0.md

```diff
@@ -31,95 +31,95 @@ changelog: |
     (Guillaume Munch-Maccagnoni, review by Anil Madhavapeddy and KC
      Sivaramakrishnan)
   
   - [#12875](https://github.com/ocaml/ocaml/issues/12875), [#12879](https://github.com/ocaml/ocaml/issues/12879), [#12882](https://github.com/ocaml/ocaml/issues/12882): Execute preemptive systhread switching as a
     delayed pending action. This ensures that one can reason within the
     FFI that no mutation happens on the same domain when allocating on
     the OCaml heap from C, consistently with OCaml 4. This also fixes
-    further bugs with the multicore systhreads implementation.
+    further bugs with the Multicore systhreads implementation.
     (Guillaume Munch-Maccagnoni, bug reports and suggestion by Mark
      Shinwell, review by Nick Barnes and Stephen Dolan)
   
   - [#12408](https://github.com/ocaml/ocaml/issues/12408): `Domain.spawn` no longer leaks its functional argument for
-    the whole duration of the children domain lifetime.
+    the whole duration of the children's domain lifetime.
     (Guillaume Munch-Maccagnoni, review by Gabriel Scherer)
   
   - [#12409](https://github.com/ocaml/ocaml/issues/12409): Fix unsafety and deadlocks should an asynchronous exception
     arise at specific locations during domain creation and shutdown.
     (Guillaume Munch-Maccagnoni, review by Gabriel Scherer)
   
-  - [#11911](https://github.com/ocaml/ocaml/issues/11911), [#12381](https://github.com/ocaml/ocaml/issues/12381): Restore statmemprof functionality in part, with
-    some API changes in Gc.Memprof.
+  - [#11911](https://github.com/ocaml/ocaml/issues/11911), [#12381](https://github.com/ocaml/ocaml/issues/12381): Restore `statmemprof` functionality in part, with
+    some API changes in `Gc.Memprof`.
     (Nick Barnes, review by Jacques-Henri Jourdan
      and Guillaume Munch-Maccagnoni).
   
-  - [#12430](https://github.com/ocaml/ocaml/issues/12430): Simplify dynamic bytecode loading in Meta.reify_bytecode
+  - [#12430](https://github.com/ocaml/ocaml/issues/12430): Simplify dynamic bytecode loading in `Meta.reify_bytecode`
     (Stephen Dolan, review by Sébastien Hinderer, Vincent Laviron and Xavier
      Leroy)
   
-  - [#12489](https://github.com/ocaml/ocaml/issues/12489): Fix an error-handling bug in caml_alloc_sprintf
+  - [#12489](https://github.com/ocaml/ocaml/issues/12489): Fix an error-handling bug in `caml_alloc_sprintf`
     (Stephen Dolan, report by Chris Casinghino, review by Jeremy Yallop
      and Xavier Leroy)
   
   - [#11307](https://github.com/ocaml/ocaml/issues/11307): Finish adapting the implementation of asynchronous actions for
-    multicore: soundness, liveness, and performance issues.
+    Multicore: soundness, liveness, and performance issues.
     Do not crash if a signal handler is called from an unregistered C
-    thread, and other possible soundness issues. Prevent issues where join
+    thread and other possible soundness issues. Prevent issues where join
```

Comment:
It is probably indeed  `joins`

Reply:
Is "joins" a thing on other domains? 
Prevent issues where "joins" on other domains could make the toplevel not responsible to Ctrl-C.
... or should it be: Prevent issues where when it joins other domains. it could make the toplevel not responsible to Ctrl-C.

Also "not responsible" or "unresponsive" ?

Reply:
Yes, it is a `join`: the act of waiting for forked threads of computations to join back into a single point.
`unresponsive` was probably the intended term.

Reply:
Thanks! Changed!